quinn-proto/src/connection/mod.rs

```diff
@@ -362,6 +435,85 @@ impl Connection {
         this
     }
 
+    /// Set up qlog for this connection.
+    #[cfg(feature = "qlog")]
+    pub fn set_qlog(
+        &mut self,
+        writer: Box<dyn std::io::Write + Send + Sync>,
+        title: String,
+        description: String,
```

Comment:
Should these be `Option`? They don't seem to be required by the crate, and `None` is probably more convenient for a lot of quick and dirty stuff.